<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnOpenOrdersFile">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>10</y>
     <width>88</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leOrdersFilePath">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>481</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>订单座位csv文件</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>区域座位csv文件</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leAreaSeatsFilePath">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>481</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpenAreaSeatsFile">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>50</y>
     <width>88</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStartArrangeSeats">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>10</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>开始排座</string>
   </property>
  </widget>
  <widget class="LogTextEdit" name="teLog">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>1091</width>
     <height>621</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LogTextEdit</class>
   <extends>QTextEdit</extends>
   <header>logtextedit</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
